homeassistant/components/withings/sensor.py

```diff
@@ -86,14 +89,17 @@ class WithingsMeasurementSensorEntityDescription(
         measurement_type=MeasurementType.MUSCLE_MASS,
         translation_key="muscle_mass",
         native_unit_of_measurement=UnitOfMass.KILOGRAMS,
+        suggested_display_precision=2,
         device_class=SensorDeviceClass.WEIGHT,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     WithingsMeasurementSensorEntityDescription(
         key="bone_mass_kg",
         measurement_type=MeasurementType.BONE_MASS,
         translation_key="bone_mass",
+        icon="mdi:bone",
```

Comment:
Not sure if this was intended or is bonus code

Reply:
I can add it in the PR description, but it was too small to create a separate PR imo

Reply:
technically should be separate PRs, but not likely worth the churn and its small enough to put in there so 🤷

Reply:
I also expected mdi had more icons for stuff like blood pressure and muscles, so I expected I could do one PR where I just add more icons in general, but apparently they don't have them.